actions/DisplayAction.php

```diff
@@ -153,7 +153,7 @@ public function execute() {
 					'donationUri'  => $bridge->getDonationURI(),
 					'icon' => $bridge->getIcon()
 				);
-			} catch(Error $e) {
+			} catch(\Throwable $e) {
```

Comment:
Comparing the two branches, there are two differences. Might be nice to mention those in the commit message.

1. The `Error` branch did not cache the data. Now it does. I presume that is fine since `Error`s will likely be more permanent than `Exception`s and we were caching the latter.

Reply:
None of the catch statements did any kind of caching.

Reply:
You are right, I must have gotten confused by indentation levels when copying it into Meld.